tests/e2e/backup_restore_suite_test.go

```diff
@@ -300,7 +301,7 @@ func tearDownBackupAndRestore(brCase BackupRestoreCase, installTime time.Time, r
 
 	err = dpaCR.Delete(runTimeClientForSuiteRun)
 	gomega.Expect(err).ToNot(gomega.HaveOccurred())
-	gomega.Eventually(lib.IsNamespaceDeleted(kubernetesClientForSuiteRun, brCase.Namespace), timeoutMultiplier*time.Minute*2, time.Second*5).Should(gomega.BeTrue())
+	gomega.Eventually(lib.IsNamespaceDeleted(kubernetesClientForSuiteRun, brCase.Namespace), timeoutMultiplier*time.Minute*10, time.Second*5).Should(gomega.BeTrue())
```

Comment:
could be 5 as in here https://github.com/openshift/oadp-operator/pull/1381/files#diff-c3773538f578442b3a133ab0748f951d2edb007bd5e9a7d0d80091d07e98464bR126 ?

Reply:
Sure, done.